effects/timed_areas.lua

```diff
@@ -0,0 +1,798 @@
+------ Timed Areas CEGs
+------ This script produces two-ish types of CEGs, each following a naming convention:
+--
+-- explosion_generator := <damage_type>-area-<radius>-<lifetime>
+--
+-- where   damage_type := acid | fire | ...
+--         radius      := <number : 000>
+--         lifetime    := dur-<frames> | <repetition>
+--         frames      := <number : 000>
+--         repetition  := looped | brief
+--
+-- Ex: "acid-area-200-dur-300" is an ongoing acid pool and "fire-area-37-looped" is a recurring burn effect.
+--
+-- Our general purpose is to maintain standard areas with a preset radius and lifetime in frames.
+--
+-- This could be used to move all visual effects to the explosion generators on area weapons, which then means
+-- areas can be suppressed (by whatever various effect) and properly terminate their CEGs early. Whether that
+-- is ever a thing you need to do, though, is a different story.
+
+-- local gameSpeed  = Game.gameSpeed -- lmao wrecked
+local gameSpeed  = 30
```

Comment:
The line causes a crash.

I don't think any system tables are accessible in the expgen def files. There's a code comment strongly advising against post-processing for explosion defs, too, so I didn't investigate that much, but if anything needed to be adjusted to a global, setting, or preference, I think it would be there. Explosion generators have these values hard-coded.